deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/workspace/DeegreeWorkspaceInitializer.java

```diff
@@ -175,13 +181,14 @@ private void initOafDatasets( Workspace newWorkspace ) {
             String id = resourceResourceIdentifier.getId();
             OafResource resource = (OafResource) newWorkspace.getResource( OgcApiProvider.class, id );
             OafDatasetConfiguration oafDatasetConfiguration = resource.getOafConfiguration();
+            addApiVersionToHeader = resource.isAddApiVersionToHeader();
```

Comment:
This is kind of a mismatch here, because as far as I can tell the `DeegreeWorkspaceInitializer` is an instance for the whole workspace but the configuration is done at the level of a dataset.

I agree that is probably desired to have a global configuration for this (I don't think there are use cases to have it enabled for one dataset but not another), but then the configuration should not be done at dataset level.

Options I see are:
1. introduce a new configuration file with global settings
2. extend an existing configuration file that has global settings in the workspace (but I don't see anything where I think it would be a good match)
3. use a simpler mechanism for global configuration (e.g. environment variables)

I think it makes sense to ask in https://github.com/deegree/deegree-ogcapi/issues/74 about any preferences on this from the deegree team.

My personal preference would be number 3, because it's really simple, or 2. if there is an existing candidate that matches well and where the schema is defined within this project.

On any of these options, if they are acceptable for the deegree team, it would be good to know any suggestions on implementation details, e.g. in case of number 1, which folder and file name to use for a new global configuration file.